From 3b86722e504c686d3e825ca8870708cce7580190 Mon Sep 17 00:00:00 2001
From: Pavel Raiskup <praiskup@redhat.com>
Date: Mon, 27 Jun 2016 23:10:12 +0200
Subject: [PATCH] tests: fix bug in dejagnu testcase

This resolves testsuite error:
ERROR: tcl error sourcing ./spanner.test/spanner.exp.
ERROR: couldn't execute "./spanner": no such file or directory
    while executing
"spawn $SPANNER"
    (file "./spanner.test/spanner.exp" line 2)
    invoked from within
"source ./spanner.test/spanner.exp"
    ("uplevel" body line 1)
    invoked from within
"uplevel #0 source ./spanner.test/spanner.exp"
    invoked from within
"catch "uplevel #0 source $test_file_name""

.. which was there probably for a very long time, but now with
dejagnu 1.6 the runtest binary started to validly exit with
non-zero status.

* t/check12.sh (Makefile.am): Also distribute files from
$(DEJATOOL) because otherwise 'runtest' binary fails later during
'make distcheck' because of incomplete list of distributed files.

Proposed upstream:
http://thread.gmane.org/gmane.comp.sysutils.automake.patches/9941

---
 t/check12.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/t/check12.sh b/t/check12.sh
index 0f3928a..306c720 100644
--- a/t/check12.sh
+++ b/t/check12.sh
@@ -61,6 +61,7 @@ DEJATOOL = hammer spanner
 AM_RUNTESTFLAGS = HAMMER=$(srcdir)/hammer SPANNER=$(srcdir)/spanner
 EXTRA_DIST += hammer.test/hammer.exp
 EXTRA_DIST += spanner.test/spanner.exp
+EXTRA_DIST += $(DEJATOOL)
 END
 
 cat > hammer << 'END'
-- 
2.7.4

